package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "epam-node-js-training",
+  "version": "1.0.0",
+  "description": "Epam node js training",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint src/",
+    "build": "babel ./src -d dist",
+    "task1": "npm run lint && npm run build && nodemon ./dist/task1/stringReverse",
```

Comment:
It is also possible to use https://babeljs.io/docs/en/next/babel-node.html it automatically builds so you do not have to build it manually.

Reply:
Thanks for suggection. Done